### PR TITLE
[bitnami/osclass] Add VIB tests

### PR DESCRIPTION
**Pull request by marcosbc**
### Description of the change

The main objective of this PR is to publish our Bitnami OsClass container using VMware Image Builder. In order to do that, several changes are included:

* Increasing the existing test coverage of the asset by adding Goss tests.
* Update verify and publish VIB pipeline's definitions.

### Benefits

* Ensuring higher quality of the container catalog.
* Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could introduce additional flakiness to the CI/CD.

### Applicable issues

N/A

### CODE REVIEW

File: .vib/osclass/goss/osclass.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+command:
+  check-app-version:
+    exec: grep "'version'\s*=>" /opt/bitnami/osclass/oc-content/languages/en_US/index.php
+    exit-status: 0
+    stdout:
+      - /'version'\s*=> '{{ .Env.APP_VERSION }}',/
```
Comment (joancafom) [inline .vib/osclass/goss/osclass.yaml:6]: Checking the version is enough (and also, the interesting part for us)

```suggestion
      - "{{ .Env.APP_VERSION }}"
```

File: .vib/osclass/goss/vars.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - render-template
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/osclass
+      - /opt/bitnami/osclass
+      - /opt/bitnami/osclass/oc-content
+files:
+  - paths:
+      - /opt/bitnami/osclass/config-sample.php
```
Comment (joancafom) [inline .vib/osclass/goss/vars.yaml:14]: I would not check this file here as we have not performed any relevant operation to it. This is something that should come for granted in compilation in my opinion.

File: .vib/osclass/goss/goss.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+gossfile:
+  # Goss tests exclusive to the current container
+  ../../osclass/goss/osclass.yaml: {}
+  # Load scripts from .vib/common/goss/templates
```
Comment (joancafom) [inline .vib/osclass/goss/goss.yaml:4]: A new template ([check-apache-libphp.yaml](https://github.com/bitnami/containers/blob/main/.vib/common/goss/templates/check-apache-libphp.yaml)) has been recently added that I think would perfectly fit here. Could you please add it?

**Review by joancafom — COMMENTED** [review]
Thanks for your PR! I left you some comments as a review, take a look at it whenever you have a chance 😁

File: .vib/osclass/goss/vars.yaml (hunk 2 above)
Comment (marcosbc) [reply .vib/osclass/goss/vars.yaml:14]: Well, I think that we shouldn't take anything for granted. But it's true that chart tests won't work if this file does not exist, so I'll remove it.

File: .vib/osclass/goss/goss.yaml (hunk 3 above)
Comment (marcosbc) [reply .vib/osclass/goss/goss.yaml:4]: Added!

File: .vib/osclass/goss/osclass.yaml (hunk 1 above)
Comment (marcosbc) [reply .vib/osclass/goss/osclass.yaml:6]: Done!

**Review by joancafom — COMMENTED** [review]
Please, remove the trigger in `docker-compose.yml` and I'll happily accept the PR. Your changes LGTM 🚀

**Review by joancafom — APPROVED** [review]

**Pull request merged**